src/Redirected.php

```diff
@@ -19,7 +19,7 @@ interface Redirected
      *
      * The body of those responses is not available.
      *
-     * @return list<PartialResponse>
+     * @return array<PartialResponse>
```

Comment:
Why?
`list` it is psalm type for numeric array (where keys started from 0 and incrementally increases)